README.md

```diff
@@ -79,6 +79,12 @@ rasa test core --stories test/test_stories.md
 
 `config.yml` - training configurations for the NLU pipeline and policy ensemble
 
+## Action Server
```

Comment:
I like this description a lot, but having a `Action Server` section by itself does not help I think. I also think that we should store our version of `docker-compose.override.yml` in this repo, off course with the actual keys removed. 

I propose that you remove this section from the current README, but then create a separate issue for an enhancement to the README to include a section `How to deploy Sara to a server using Rasa X`, and that we then add these things like including the `docker-compose.override.yml` as part of that new issue.

Reply:
To make things clearer in the current README that we only cover installation and running locally, you could update the titles and descriptions of those sections:

## Installation locally
To install Sara locally, ...

## To run Sara locally
...